config.yaml.example

```
@@ -0,0 +1,2 @@
+username: username-config
+password: password-config
```

```suggestion
username: mySolidfireUsername
password: mySolidfirePassword
```

We probably also want to show _all_ of the configurable params in this example config file here